### PR TITLE
#5 Add Unordered List of Items with a asterisk (*) format

### PR DESCRIPTION
**Pull request by Sachin-chaurasiya**

### CODE REVIEW

**Comment by Sachin-chaurasiya** [comment]
@atapas, Please review.

File: README.md [hunk 1]
```diff
@@ -76,6 +76,20 @@ Many Thanks to all the `Stargazers` who has supported this project with stars(
 - Tea
 - Beer
```
Comment (atapas) [inline README.md:78]: @Sachin-chaurasiya Can we please add a line here explaining the below one is an alternate syntax?

**Review by atapas — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (Sachin-chaurasiya) [reply README.md:78]: @atapas , Yes, I will add that.

Comment (Sachin-chaurasiya) [reply README.md:78]: @atapas , I have made the changes, please review.

**Review by atapas — APPROVED** [review]
LGTM

**Pull request merged**